src/icons/folderIcons.ts

```diff
@@ -10,18 +10,17 @@ export const folderIcons: FolderTheme[] = [
         rootFolder: { name: 'folder-root' },
         icons: [
             { name: 'folder-src', folderNames: ['src', 'source', 'sources'] },
-            { name: 'folder-dist', folderNames: ['dist', 'out', 'build', 'release', 'bin'] },
+            { name: 'folder-dist', folderNames: ['dist', 'out', 'build', 'release', 'bin', 'ltximg', 'latex.out'] },
             {
                 name: 'folder-css',
                 folderNames: ['css', 'stylesheet', 'stylesheets', 'style', 'styles']
             },
             { name: 'folder-sass', folderNames: ['sass', '_sass', 'scss', '_scss'] },
-            {
-                name: 'folder-images',
-                folderNames: ['images', 'image', 'img', 'icons', 'icon', 'ico', 'screenshot', 'screenshots']
-            },
+            { name: 'folder-images', folderNames: ['images', 'image', 'img'] },
+            { name: 'folder-icon', folderNames: ['icons', 'icon', 'ico'] },
+            { name: 'folder-screenshot', folderNames: ['screenshot', 'screenshots'] },
             { name: 'folder-scripts', folderNames: ['script', 'scripts'] },
-            { name: 'folder-node', folderNames: ['node_modules'] },
+            { name: 'folder-node', folderNames: ['node_modules', 'npm'] },
```

Comment:
`node !== npm`

I think the `node` folder icon does not fit very good to the `npm` folder. And I haven't seen so many `npm` folders before, not sure if we need an icon for that folder at all.

Reply:
you're right